packages/components/tests/e2e/specs/appmap/componentDiagram.spec.js

```diff
@@ -156,10 +156,7 @@ context('AppMap component diagram', () => {
 
       // Go back to the component diagram
       cy.get('.tabs .tab-btn').first().click();
-      cy.get(`.node[data-id="HTTP server requests->GET /admin/orders"]`).should(
-        'have.class',
-        'highlight'
-      );
+      cy.get(`.node[data-id="GET /admin/orders"]`).should('have.class', 'highlight');
```

Comment:
I don't think this is important at the moment, but these attributes should probably switch to use FQID in the future.